stl/inc/atomic

```diff
@@ -2803,9 +2804,17 @@ _EXPORT_STD struct atomic_flag { // flag with test-and-set semantics
 #endif // _HAS_CXX20
 
 #if 1 // TRANSITION, ABI
-    atomic<long> _Storage;
+    atomic<long> _Storage
+#if !_HAS_CXX20 && defined(__EDG__)

```

Comment:
We need a `TRANSITION` comment here citing a report for the compiler bug this is working around. (Two places.)

Reply:
Oh, the changed doesn't seem right. The error seems caused by inherited non-template trivial default constructors. I need to find out how to not inherit them.